simulation_ws/src/turtlebot3_description_reduced_mesh/CMakeLists.txt

```diff
@@ -1,13 +1,20 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(turtlebot3_description_reduced_mesh)
```

Comment:
Let's remove turtlebot3_description_reduced_mesh from this repository and use the one in https://github.com/aws-robotics/turtlebot3-description-reduced-mesh (ros2 branch). Add it to the .rosinstall like so: https://github.com/aws-robotics/aws-robomaker-sample-application-voiceinteraction/blob/ros2-devel/simulation_ws/.rosinstall#L2

Reply:
Yes, done

Reply:
Removing `turtlebot3_description_reduced_mesh` from this repository led to the changes in the small_house_turtlebot_navigation.launch.py in the` aws-robomaker-small-house-world`. How to merge these changes? Should I need to open an issue or raise PR  for `aws-robomaker-small-house-world` on ros2 branch?

Reply:
Raised PR on ros2 branch for `aws-robomaker-small-house-world `, please review for the same.